pkg/controllers/bootstrap/crds.go

```diff
@@ -0,0 +1,129 @@
+// Copyright Contributors to the Open Cluster Management project
+package bootstrap
+
+import (
+	"context"
+	"embed"
+	"time"
+
+	"github.com/openshift/library-go/pkg/assets"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+
+	"k8s.io/apiextensions-apiserver/pkg/apihelpers"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+
+	"open-cluster-management.io/multicluster-controlplane/pkg/util"
+)
+
+var baseCRDs = []string{
+	"crds/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml",
+	"crds/0000_00_clusters.open-cluster-management.io_managedclustersets.crd.yaml",
+	"crds/0000_01_clusters.open-cluster-management.io_managedclustersetbindings.crd.yaml",
+	"crds/0000_00_work.open-cluster-management.io_manifestworks.crd.yaml",
+	"crds/0000_00_addon.open-cluster-management.io_clustermanagementaddons.crd.yaml",
```

Comment:
do we really need `clustermanagementaddon` and `managedclusteraddon` now?

Reply:
for ocm part, I think we need